### DISCUSSION TITLE
Using sideband with the Heltec ESP32 V3 board

### DISCUSSION THREAD

**Discussion by ThomasFreedman** [markqvist/Reticulum#488] [Sideband]
Is this possible? If so I presume it must be flashed with different firmware than Meshtastic, or not?

Is there a guide on how to use these SX1262 devices with sideband?

**Comment by cobraPA** [comment]
Support is just being added to the RNode firmware for the Heltec V3, there is a pull request to update the main firmware.
https://github.com/markqvist/RNode_Firmware/pull/73
Did you mean RNode?  Sideband does not use the Meshtastic firmware.  The pull request supports RNode use over serial connections with Sideband and Heltec V3 devices.

**Reply by ThomasFreedman** [reply]
You confirmed that Reticulum / Sideband wouldn't work with Meshtastic firmware and won't work with the Heltec V3 right now, but will when / if the pull request to support it is implemented which seems likely. 

I'm new to LoRa. Just bought a pair of Heltec V3 boards with cases & antennas. I loaded the Meshtastic firmware and they appear to be working. Will ditch Meshtastic FW for RNode when it's available for V3.

Nice to see Erethon's work on the web flasher to flash RNode firmware. Hope to see support for the Heltec V3 added there too.
